server_versions.json

```diff
@@ -182,28 +146,19 @@
     "name": "Vanilla",
     "type": "SERVER",
     "latestVersion": "1.20.6",
-    "latestSnapshot": "24w14a",
+    "latestSnapshot": "24w18a",
     "downloadLinks": [
       {
         "version": "1.20.6",
         "link": "https://piston-data.mojang.com/v1/objects/145ff0858209bcfc164859ba735d4199aafa1eea/server.jar"
       },
-      {
-        "version": "1.20.5",
-        "link": "https://piston-data.mojang.com/v1/objects/79493072f65e17243fd36a699c9a96b4381feb91/server.jar"
-      },
       {
         "version": "1.20.4",
         "link": "https://piston-data.mojang.com/v1/objects/8dd1a28015f51b1803213892b50b7b4fc76e594d/server.jar"
       },
       {
-        "version": "24w14a",
-        "link": "https://piston-data.mojang.com/v1/objects/960cb0e5c794b02abdbcdbdc15b4de058b222118/server.jar",
-        "isSnapshot": true
-      },
-      {
-        "version": "24w14potato",
```

Comment:
Why did you remove this? I would leave it there

Reply:
april is over and i think its better to remove it, because nobody more plays the april version